cluster-scope/base/batch/cronjobs/service-catalog-k8s-plugin/cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: upload-token-to-vault
+spec:
+  schedule: "0 * * * *"
```

Comment:
I think you should also set:
- https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy
- https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#jobs-history-limits
- https://kubernetes.io/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically

Reply:
I've set it so that can view the last 3 failed Jobs and if the job is successful it gets deleted immediately. Let me know if thats fine.